lib/bank_account/saving.rb

```diff
@@ -0,0 +1,45 @@
+module BankAccount
+	
+
+	class Saving < Account
+
+		def initialize(id, initial_balance)
+			super	
+			if @initial_balance < 10 
+				raise ArgumentError, "an error has occurred. Balance must equal more than $10. Please make a deposit."
+			end	
+			@available_balance
+		end #initialize end
+
+
+		def withdraw(w_amount)  
+			trans_fee = 2
+			mandatory_balance = 10
+			if @current_balance < 12
```

Comment:
One thing to consider is doing a bit less hard coding of values. 
Can you use some of the variables you've already created instead of using the number `12`? If so then if the fees or mandatory balance requirements change later you have fewer pieces of code you need to change which minimizes the potential for bugs. :)